base/ltcmdhooks.dtx

```diff
@@ -1394,3 +1394,11 @@
 %
 % \Finale
 %
+
+
+\endinput
+
+%%% Local Variables: 
+%%% mode: latex
+%%% coding: utf-8-unix
+%%% End: 
```

Comment:
Stray local lines?

Reply:
no not really. I need them because the files  contain characters that confuse emails and the file is not opened as utf8 but some strange mixed encoding. They are behind \endinput, so harmless but I coule alternatively, write them as
```
^^A  Needed for emacs
^^A
^^A  Local Variables: 
^^A  mode: latex
^^A  coding: utf-8-unix
^^A  End: 
```
if that is preferred --- I seem to need them in my setup.